gym-unity/setup.py

```diff
@@ -1,21 +1,15 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
-from os import path
-
-here = path.abspath(path.dirname(__file__))
-
-with open(path.join(here, "..", "VERSION")) as f:
-    version_string = f.read().strip()
 
 setup(
     name="gym_unity",
-    version=version_string,
+    version="0.4.7",
```

Comment:
Should we just tick this to 0.4.8?

Reply:
Hrm... I'd rather just undo the old PR, and make any other changes separately.

Reply:
OK, I'll re-open that PR after this one